core/keys/punctuation.talon-list

```diff
@@ -0,0 +1,29 @@
+list: user.punctuation
```

Comment:
This probably needs to be kept in python or moved to a csv for compatibility with create_spoken_forms